### DISCUSSION TITLE
Is it possible to embed a Snapshot proposal into a 3rd party app today?

### DISCUSSION THREAD

**Discussion by maxjsegall** [snapshot-labs/snapshot#3475] [Q&A]
I'm building a new app, and I'd like to embed a token-gated Snapshot vote into the app directly. Is this possible today?

**Comment by ChaituVR** [comment]
You can try using our SDK https://docs.snapshot.org/snapshot.js in your app 
Or you can use a custom domain with our whitelabel solution check https://docs.snapshot.org/spaces/add-custom-domain
